src/components/Header/ContactModal/ContactOtpForm.tsx

```diff
@@ -0,0 +1,73 @@
+import { FormControl } from "@chakra-ui/react"
+import {
+  Button,
+  FormErrorMessage,
+  FormLabel,
+  Input,
+} from "@opengovsg/design-system-react"
+import { useEffect } from "react"
+import { useForm } from "react-hook-form"
+
+export interface ContactOtpProps {
+  otp: string
+}
```

Comment:
Should this be part of `types/contact.ts`? Then `VerifyOtpParams` can be modified to be a combination of both the `mobile` and `otp` keys.

Reply:
Hmm I'm thinking that `ContactOtpProps` belongs in this component because it's only being used in the form here - types/contact are being used in the useUpdate hooks and ContactService, which is why I thought it was better to separate those out into a separate type

Reply:
Ahh I was thinking more because the props are very similar in this case (both having the otp string). So types/contact.ts will have 3 types in total and can show the relationship between those 3 types.

Reply:
changed in https://github.com/isomerpages/isomercms-frontend/pull/1092/commits/ec09cc10e0938934c5fbdbd75b4119fb05a25a43!